search_gov_crawler/search_gov_spiders/pipelines.py

```
@@ -13,6 +13,12 @@
 from search_gov_crawler.search_gov_spiders.items import SearchGovSpidersItem
 from search_gov_crawler.elasticsearch.es_batch_upload import SearchGovElasticsearch
 
+def safe_del(item, key: str):
+    try:
+        del item[key]
+    except Exception as _:
```

Can you add a docstring to `safe_del`? 

Are you meaning to catch all Exceptions or just KeyError?  Why would the item not have the key?  Just making it safe for any delete operation?

---

All exceptions, since the item might also be None. This should never happen, but in the off chance we process the same item again which will not have those keys

---

@selfdanielj Added the requested doc string